src/reducers/sortings.ts

```diff
@@ -68,7 +68,7 @@ const isRemoveUnitLabelAction = (x: any): x is RemoveUnitLabelAction => (
 )
 
 export type State = Sorting[]
-export type Action = AddSortingAction | DeleteSortingsAction | DeleteAllSortingsForRecordingsAction | SetSortingInfoAction | AddUnitLabelAction | RemoveUnitLabelAction | SetExternalSortingUnitMetricsAction
+export type Action = (AddSortingAction | DeleteSortingsAction | DeleteAllSortingsForRecordingsAction | SetSortingInfoAction | AddUnitLabelAction | RemoveUnitLabelAction | SetExternalSortingUnitMetricsAction) & {persistKey?: string}
```

Comment:
What's `persistKey` about?

Reply:
that's the system for sending dispatched actions to the feed prior to having them act on the state.